src-electron/rest/user-data.js

```diff
@@ -108,7 +109,8 @@ function httpGetPackageNotifications(db) {
 function httpGetPackageNotificationsByPackageId(db) {
   return async (request, response) => {
     let packageId = request.params.packageId
-    let notifications = await queryPackageNotification.getNotificationByPackageId(db, packageId)
+    let notifications =
+      await queryPackageNotification.getNotificationByPackageId(db, packageId)
```

Comment:
Why do we have these formatting diffs? Do we not all use the same formatter?

Reply:
not sure